extensions/redis-client/runtime/src/main/java/io/quarkus/redis/client/RedisClientName.java

```diff
@@ -0,0 +1,40 @@
+package io.quarkus.redis.client;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+import javax.inject.Qualifier;
+
+/**
+ * Marker annotation to select the redis client
```

Comment:
```suggestion
 * Marker annotation to select the Redis client.
```
(it's minor but this is public API so let's get it right)